### PR TITLE
fix: yarn - do not update checksum by default

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

This PR disables Yarn's automatic checksum updates to enhance security and maintain consistency. It ensures that dependencies remain unchanged unless explicitly intended, preventing potential unauthorized or accidental modifications. This is crucial for a stable and secure build process.

### Type of change

Bug fix

### Tests

No test

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 nodeLinker: node-modules
 
-checksumBehavior: 'update'
+# checksumBehavior: 'update'
```
Comment (pwltr) [inline .yarnrc.yml]: can remove the comment

**Review by pwltr — APPROVED** [review]

**Comment by pwltr** [comment]
@limpbrains Why does the `react-native-keychain` have a different checksum for you than me? I can't install it

**Comment by limpbrains** [comment]
I was afraid of that. Same issue blocked BW migration to yarn v3.
We might want to revert it

**Comment by pwltr** [comment]
meh, looks like packages pulled from git can have slightly different content depending on OS when building with yarn v3. Yeah let's revert.

**Pull request merged**